printnames.py

```diff
@@ -0,0 +1,12 @@
+class PrintNames:
+
+    def __init__(self, first_name, last_name):
+        self.first_name = first_name
+        self.last_name = last_name
+        print("Full Name is {0} {1}".format(self.first_name[-1::-1], self.last_name[-1::-1]))
```

Comment:
Don't print here. Add code to return this when the object is printed.

Reply:
done